src/sql/workbench/contrib/query/browser/queryResultsView.ts

```diff
@@ -398,18 +417,29 @@ export class QueryResultsView extends Disposable {
 	}
 
 	public showPlan(xml: string) {
-		this.input?.state.visibleTabs.add(this.qpTab.identifier);
-		if (!this._panelView.contains(this.qpTab)) {
-			this._panelView.pushTab(this.qpTab);
-		}
-		this.input?.state.visibleTabs.add(this.topOperationsTab.identifier);
-		if (!this._panelView.contains(this.topOperationsTab)) {
-			this._panelView.pushTab(this.topOperationsTab);
-		}
+		// this.input?.state.visibleTabs.add(this.qpTab.identifier);
```

Comment:
Shouldn't this be left as is?

Reply:
oops, this was commented out so only one pane showed up during internal demo.  this code should be commented out.